client/app/scripts/superdesk-search/views/search-results.html

```diff
@@ -17,46 +21,16 @@
             <div sd-multi-action-bar></div>
         </header>
 
-
-
         <div class="content" sd-updown data-items="items._items" data-select="preview(item)">
             <div class="loading-indicator" ng-show="loading" translate>loading</div>
 
             <div sd-search-tags></div>
 
-            <div sd-shadow>
-                <ul class="{{view}}-view list-view">
-                    <li ng-repeat="item in items._items" class="list-item-view" ng-class="{active: item === selected.preview || item._id === selected.fetch}">
-                        <div sd-media-box></div>
-                    </li>
-                </ul>
-            </div>
-        </div>
-    </div>
-
-    <div class="preview-pane content-item-preview" ng-if="!selected.fetch" ng-init="tab = 'content'">
-        <header>
-            <ul class="nav nav-tabs">
-                <li ng-class="{active: tab === 'content'}">
-                    <button ng-click="tab = 'content'"><span translate>Content</span></button>
-                </li>
-                <li ng-class="{active: tab === 'metadata'}">
-                    <button ng-click="tab = 'metadata'"><span translate>Metadata</span></button>
-                </li>
-                <li ng-class="{active: tab === 'related'}">
-                    <button ng-click="tab = 'related'"><span translate>Related Items</span></button>
+            <ul class="{{view}}-view list-view">
+                <li ng-repeat="item in items._items" class="list-item-view" ng-class="{active: item === selected.preview || item._id === selected.fetch}">
```

Comment:
Would it be possible to perhaps join two fields to obtain a unique combination?
If not, well, then tough luck I guess...